packagingbuild/centos7/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN yum -y install \
     git \
     mercurial \
     openssh \
-    subversion
+    subversion \
+    setup
```

Comment:
Don't think we need this rpm dependency during the st2 Package build (`packagingbuild`).
However we'll need it during the st2 Package tests (`packagingtest`).